pkg/splunk_logger/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/osbuild/osbuild-composer/pkg/splunk_logger
+
+go 1.20
```

Comment:
Is it a good idea to have a different go version here? The main package is still on 1.19

Reply:
Good point, makes sense to stay on the same version.